frontend/src/sagas.js

```diff
@@ -481,8 +481,8 @@ export function* editProjectSaga({ projectId, changes, deletedProjectedBudgets =
     // const { deletedProjectedBudgets = [], projectedBudgets = [], ...rest } = changes;
     const { projectedBudgets = [], ...rest } = changes;
 
-    if (Object.values(rest).some(value => value !== undefined)) {
-      yield callApi(api.editProject, projectId, rest);
+    for (const budget of deletedProjectedBudgets) {
+      yield callApi(api.deleteProjectBudgetProjected, projectId, budget.organization, budget.currencyCode);
     }
```

Comment:
i change the order in the saga:
then: 
set new budgets, update budgets, delete budgets
now:
delete budgets, update budgets, set new budgets